.gitlab/specs.yml

```diff
@@ -9,7 +9,7 @@
 .gcc_mvapich2:
   variables:
     SPEC: 'gcc@$GCC_VERSION^mvapich2'
-    EXTRA_CMAKE_ARGS: '-DENABLE_DOCS=On -DENABLE_WARNINGS_AS_ERRORS=On'
+    EXTRA_CMAKE_ARGS: '-DENABLE_DOCS=On -DENABLE_WARNINGS_AS_ERRORS=On -DENABLE_DEV_BUILD=On'
```

Comment:
Could we move this to a different config, we build `gcc_mvapich2` as our release build so testing for the dev build with it isn't ideal. Maybe `clang_mavapich2`, or `gcc_spectrum`?